airbyte-integrations/connectors/source-iterable/acceptance-test-config.yml

```diff
@@ -1,27 +1,100 @@
 # See [Connector Acceptance Tests](https://docs.airbyte.com/contributing-to-airbyte/building-new-connector/connector-acceptance-tests.md)
 # for more information about how to configure these tests
 connector_image: airbyte/source-iterable:dev
-tests:
+test_strictness_level: "high"
+acceptance_tests:
   spec:
-    - spec_path: "source_iterable/spec.json"
+    tests:
+      - spec_path: "source_iterable/spec.json"
   connection:
-    - config_path: "secrets/config.json"
-      status: "succeed"
-    - config_path: "integration_tests/invalid_config.json"
-      status: "failed"
+    tests:
+      - config_path: "secrets/config.json"
+        status: "succeed"
+      - config_path: "integration_tests/invalid_config.json"
+        status: "failed"
   discovery:
+    tests:
     - config_path: "secrets/config.json"
   basic_read:
+    tests:
     - config_path: "secrets/config.json"
-      configured_catalog_path: "integration_tests/catalog.json"
-      empty_streams: ["email_send_skip", "email_complaint"]
+      expect_records:
+        path: "integration_tests/expected_records.jsonl"
+        extra_fields: no
+        exact_order: no
+        extra_records: yes
+      empty_streams:
+        - name: "web_push_click"
+          bypass_reason: "Can not populate; need messaging service configured"
```

Comment:
This is a *lot* of bypassed streams... why can't we enable a messaging service?  Do we need a test website to install Iterable on?

Reply:
cc @YowanR

Reply:
I think these are 2 important but separate efforts. The work here is our best effort to enable high-test strictness given the current state of connectors. In Q1b, we will spend some time seeding GA connectors (based on usage) and doing this will allow us to reduce (and ideally eliminate) the list of streams that were bypassed. Do you agree, @evantahler?

Reply:
I think I would prefer that GA connectors don't have this many streams skipped.  Can we come up with a rule?  Perhaps:
* no more than 10 streams skipped?
* no more than 10% of streams skipped?

Reply:
For new connectors during migration? I think that 10% is reasonable and maybe we require a reason if we go above that?

Reply:
Evan and I had a word about this. This is what we should do going forward. 
1. Let's go ahead with this PR and have it merge for Iterable.
2. Create an issue to track the fact that we have multiple stream disable. In Q1b, we'll prioritize getting this sandbox account seeded and then re-enabling all the skip streams for high-test strictness.

@lazebnyi @artem1205 Does this make sense? Please make sense?

Reply:
Look good for me. Issue is already created - https://github.com/airbytehq/airbyte/issues/22496